### PR TITLE
Feature/new design

### PR DESCRIPTION
**Pull request by DemogorGod**
Scrapped previous designs and added new ones from the new Figma file created by Hawyar.

### CODE REVIEW

**Review by shanejearley — APPROVED** [review]

File: apps/web/src/components/Wallet.vue [hunk 1]
```diff
@@ -203,6 +204,7 @@ const {
 const { deposit, getPools, withdraw } = useSSV()
 
 onMounted(async () => {
+  console.log('got to Wallet.vue onMounted')
```
Comment (ccali11) [inline apps/web/src/components/Wallet.vue:207]: Can probs delete too.

File: apps/web/src/components/charts/LineChartJS.vue [hunk 2]
```diff
@@ -86,6 +86,7 @@ onMounted(() => {
             WIDTH = line_chart_container_el.offsetWidth
             HEIGHT = line_chart_container_el.offsetHeight
         }
+        console.log(WIDTH, HEIGHT)
```
Comment (ccali11) [inline apps/web/src/components/charts/LineChartJS.vue:89]: Delete?

File: apps/web/src/composables/router.ts [hunk 3]
```diff
@@ -39,27 +28,27 @@ const router = createRouter({
...
-        next()
-    } else if (to.fullPath === '/auth' && loggedIn) {
-        next('/')
-    } else if (!loggedIn) {
-        next('/auth')
-    } else {
-        next()
-    }
-})
+// router.beforeEach(async (to, from, next) => {
```
Comment (ccali11) [inline apps/web/src/composables/router.ts:31]: Keep this commented out for now?

Comment (ccali11) [reply apps/web/src/composables/router.ts:31]: Or maybe we don't have an auth page anymore...

File: apps/web/src/composables/users.ts [hunk 4]
```diff
@@ -36,6 +36,7 @@ export default function useUsers () {
     */
     async function checkUserSessionExists() : Promise<boolean> {
         try {
+            console.log('got to checkUserSessionExists')
```
Comment (ccali11) [inline apps/web/src/composables/users.ts:39]: Can probs delete

**Review by ccali11 — CHANGES_REQUESTED** [review]
@DemogorGod - Just take a quick look at my comments. Nothing big.

File: apps/web/src/components/Wallet.vue (hunk 1 above)
Comment (DemogorGod) [reply apps/web/src/components/Wallet.vue:207]: This is your component, do you need that log or not? @ccali11

File: apps/web/src/composables/router.ts [hunk 5]
```diff
@@ -1,27 +1,16 @@
...
 /* eslint-disable @typescript-eslint/ban-ts-comment */
-import Dashboard from '@/pages/dashboard/Dashboard.vue'
+import Overview from '@/pages/overview/Overview.vue'
 import Test from '@/pages/test/Test.vue'
-import Auth from '@/pages/auth/Auth.vue'
 
 const routes = [
     { 
         path: '/', 
-        name: Dashboard,
```
[the remark this answers is not in the document]
Comment (DemogorGod) [reply apps/web/src/composables/router.ts:13]: Not sure what you're mentioning but the standard is for it to have a name which it does.

**Review by ccali11 — APPROVED** [review]

**Pull request merged**